src/res/SfmlFileResource.hpp

```diff
@@ -1,5 +1,5 @@
-#ifndef ChessPlusPlus_Res_SfmlFileResourceClass_HeaderPlusPlus
-#define ChessPlusPlus_Res_SfmlFileResourceClass_HeaderPlusPlus
+#ifndef chesspp_res_SfmlFileResource_HeaderPlusPlus
+#define chesspp_res_SfmlFileResource_HeaderPlusPlus
 
 #include "ResourceManager.hpp"
 //#include "SFML.hpp"
```

Comment:
It's kind of a technicality - the template only works with SFML classes but it doesn't need the include until it is instantiated. What do you suggest?